config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+  namespace :api do
+    namespace :v1 do
+      get 'books', to: 'book#index'
```

Comment:
Use routing resources so as to standardize URLS. Check this doc https://guides.rubyonrails.org/routing.html